plugins/com.google.cloud.tools.eclipse.ui.util/src/com/google/cloud/tools/eclipse/ui/util/RadioSelectionListener.java

```diff
@@ -0,0 +1,31 @@
+package com.google.cloud.tools.eclipse.ui.util;
+
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.events.SelectionListener;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Control;
+
+public class RadioSelectionListener<C extends Control> implements SelectionListener {
```

Comment:
RadioSelectionListener is not used anymore, I forgot to remove it.

On Thu, Aug 11, 2016, 07:03 Elliotte Rusty Harold <notifications@github.com>
wrote:

> In
> plugins/com.google.cloud.tools.eclipse.ui.util/src/com/google/cloud/tools/eclipse/ui/util/RadioSelectionListener.java
> <https://github.com/GoogleCloudPlatform/gcloud-eclipse-tools/pull/529#discussion_r74403052>
> :
>
> > @@ -0,0 +1,31 @@
> > +package com.google.cloud.tools.eclipse.ui.util;
> > +
> > +import org.eclipse.swt.events.SelectionEvent;
> > +import org.eclipse.swt.events.SelectionListener;
> > +import org.eclipse.swt.widgets.Button;
> > +import org.eclipse.swt.widgets.Control;
> > +
> > +public class RadioSelectionListener<C extends Control> implements SelectionListener {
>
> I don't see where this class is used
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-eclipse-tools/pull/529/files/ab60fe8ba02bfe8a123a0d40213044ef4450d00b#r74403052>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADOO_vjQzyAiuBrzxiVMCoCDfGpB7R4Fks5qewF5gaJpZM4Jgjkk>
> .
>